src/common/test-runtime/project.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dependencies": {
```

Comment:
I believe this is because she is updating to the new buildtools which now has a different way of building dependencies for xunit. This is basically equivalent to what corefx has for it's test projects.

---
In reply to: [69196894](https://github.com/dotnet/buildtools/pull/850#discussion_r69196894) [](ancestors = 69196894)